### PR TITLE
SE/Reactive archetype removal

### LINKED ISSUE #6655: 4.x: SE/Reactive archetype removal

### PR DESCRIPTION
**Pull request by tvallin**
fix #6655

### CODE REVIEW

File: archetypes/helidon/src/main/archetype/mp/custom/files/src/test/java/__pkg__/TestCORS.java.mustache [hunk 1]
```diff
@@ -1,141 +1,108 @@
...
+import static org.hamcrest.Matchers.is;
 
+@HelidonTest
 class TestCORS {
 
-    private static WebClient client;
-    private static Server server;
+    @Inject
+    private WebTarget target;
+    private static final String ALLOW_RESTRICTED_HEADERS = "sun.net.http.allowRestrictedHeaders";
```
Comment (romain-grecourt) [inline archetypes/helidon/src/main/archetype/mp/custom/files/src/test/java/__pkg__/TestCORS.java.mustache:27]: Where is that coming from ?

Comment (tvallin) [reply archetypes/helidon/src/main/archetype/mp/custom/files/src/test/java/__pkg__/TestCORS.java.mustache:27]: You can see it from `BaseCrossOriginTest`. The system property is needed to be set to allow restricted header with Jersey client.

Comment (romain-grecourt) [reply archetypes/helidon/src/main/archetype/mp/custom/files/src/test/java/__pkg__/TestCORS.java.mustache:27]: How did it work without that ?

Comment (tvallin) [reply archetypes/helidon/src/main/archetype/mp/custom/files/src/test/java/__pkg__/TestCORS.java.mustache:27]: I think the reason is that reactive webClient is using `HelidonConnector` while now it is using the Jersey default `HttpUrlConnector` where this property manage restricted headers.

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**